apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -708,7 +711,10 @@ class TopInstructions extends Component {
               <DocumentationTab ref={ref => (this.documentationTab = ref)} />
             )}
             {tabSelected === TabType.REVIEW && (
-              <ReviewTab ref={ref => (this.reviewTab = ref)} />
+              <ReviewTab
+                ref={ref => (this.reviewTab = ref)}
+                onLoadComplete={this.forceTabResizeToMaxHeight}
```

Comment:
One thing to double check: If there are more comments than fill the page height, does the resize still happen in a reasonable manner?

Reply:
Good question, I'll take a look!

Reply:
Yep, so if the content of the review tab exceeds the page height, then we do end up going over the page height without a scrollbar which is a problem. I updated this so we resize to the minimum of the content height and `maxAvailableHeight`